be1-go/hub/standard_hub/message_handling.go

```diff
@@ -19,36 +19,33 @@ import (
 
 const publishError = "failed to publish: %v"
 
-// handleRootChannelPublishMesssage handles an incominxg publish message on the root channel.
+// handleRootChannelPublishMessage handles an incoming publish message on the root channel.
 func (h *Hub) handleRootChannelPublishMesssage(sock socket.Socket, publish method.Publish) error {
 	jsonData, err := base64.URLEncoding.DecodeString(publish.Params.Message.Data)
 	if err != nil {
-		err := xerrors.Errorf("failed to decode message data: %v", err)
-		sock.SendError(&publish.ID, err)
```

Comment:
I think we still need to send the error to the socket, the returned error will just be logged

Reply:
That is the thing I realized here, the Error will be sent to the socket either way this is done in the function `handleMessageFromClient` , I've had this error before that even when we get an error we receive another message afterwards this was due (like I explain in the description of this PR) to the fact that we send an error twice to the socket if an error happens on the root channel. If you take a look at how errors are handled in other channels lao,election... you will see that none of them actually send the Error to the socket but propagate it where the `handleMessageFromClient` happens and there we send the error to the socket. We could also delete the error sending there and send it beforehand but I am not sure that it will be the right approach